src/store/store.mockData.ts

```diff
@@ -0,0 +1,16 @@
+import { AppState } from 'store';
+import { REDUCER_NAME as NOTIFICCATION_REDUCER_NAME } from './notifications/reducer';
+import { notificationsReady } from './notifications/notifications.mockData';
+import { REDUCER_NAME as CURRENCY_REDUCER_NAME } from './currency/reducer';
+import { currencyReady } from './currency/currency.mockData';
+import { REDUCER_NAME as MINT_REDUCER_NAME } from './mint/reducer';
+import { mintReady } from './mint/mint.mockData';
+import { REDUCER_NAME as TRANSFER_REDUCER_NAME } from './transfer/reducer';
+import { transferReady } from './transfer/transfer.mockData';
+
+export const storeReady = {
+	[NOTIFICCATION_REDUCER_NAME]: notificationsReady,
```

Comment:
Spelling error, should be `NOTIFICATION_REDUCER_NAME` (this one is made a couple times so just `CTRL + F` them all)

Reply:
Fixed already. I really appreciate your quick reviews